### DISCUSSION TITLE
BLTOUCH always triggered

### DISCUSSION THREAD

**Discussion by chakkuchan** [Jyers/Marlin#324] [Q&A] [answered]
Just switched from Smith3D to Jyers for my E3V2, and now my BLTouch will not extend. Tried both a 5x5HS BL touch for 4.2.2 and the 15x15 UBL HS BL Touch for 4.2.2 and neither one will extend the probe. With the Smith3D this would happen occasionally and I could just manually extend the probe and restart the home process and it would work. I can't even home Z to try and do my Z offset... 

SENDING:M119
Reporting endstop status
x_min: open
y_min: open
z_min: TRIGGERED
z_probe: TRIGGERED
filament: TRIGGERED

**Comment by Jyers** [comment, marked as answer]
Is your probe connected to the BL Touch port or the z endstop pins?

**Reply by chakkuchan** [reply]
It is connected to the BL Touch Port, after switching to the newest release (2.0.17 Smith3D it was having the same problem, guessing it might be a wire issue, so reseated it and just left the Smith for now). I setup a Raspberry Pi with Octoprint since this came out, and with Octoprint the BL Touch has worked 100% of the time (that's a first regardless of firmware), so I'll try again with Jyers and UBL (since octoprint has great UBL mesh support too) and I bought a replacement wire since I can't find that molex picoblade connector that is on the end of the BL Touch.
